### ISSUE TITLE
LineLength not working with phpcbf for function argument

### ISSUE THREAD

**Issue by himanshutecstub** [squizlabs/PHP_CodeSniffer#3536]
**Describe the bug**
In my project i set Generic.Files.LineLength to 120 which is not working with Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine & Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace & phpcs

**Code sample**
```
private function newCategory($abcd, $testData, $resultData, $updateData, $dbConnection, $saveCategory, $oldCategory) {
	
}
```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<rule ref="Generic.Files.LineLength">
        <properties>
            <property name="lineLimit" value="120"/>
            <property name="absoluteLineLimit" value="120"/>
        </properties>
    </rule>
```

**To reproduce**
When try to separate each argument in new line (because shown "Line exceeds maximum limit of 120 characters" error) than return error as 
```
The closing parenthesis of a multi-line function declaration must be on a new line
(Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine)
The closing parenthesis and the opening brace of a multi-line function declaration must be on the same line
(Squiz.Functions.MultiLineFunctionDeclaration.NewlineBeforeOpenBrace)

```
Also try to fix below error using phpcbf but shown error as "FAILED TO FIX"

**Expected behavior**
Should working all rules properly

**Versions (please complete the following information):**
 - OS: Ubuntu 20
 - PHP: 7.2
 - PHPCS: 3.6.2
 - Standard:  PSR2, Squiz
